support-frontend/assets/components/countryGroupSwitcher/countryGroupSwitcher.tsx

```diff
@@ -84,7 +84,7 @@ function CountryGroupSwitcher({
 			>
 				<Menu
 					style={{
-						top: bounds.top,
+						top: bounds.top + 30,
```

Comment:
might be better to remove 30 and use bounds.bottom (add it to the bounds useState)

Reply:
We just had a catch up on this one and the `bounds` state here is used to store the outcome of the [getBoundingClientRect()](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) function. We use that function to tell us how far the 'reference' element is from the top and the left of the page so I reckon adding 30 to the resolution of bounds.top here is correct